pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -89,7 +90,8 @@ def distribution_new_customer_spend(
     def expected_new_customer_spend(self) -> xarray.DataArray:
         """Expected transaction value for a new customer"""
 
-        assert self.idata is not None, "Model must be fitted first"
+        if self.idata is None:
+            raise ValueError("Model must be fitted first")
```

Comment:
same as above

Reply:
solved in https://github.com/pymc-labs/pymc-marketing/pull/637/commits/f9dc0dc2371a3375ddc094d36d26ede25a665cf0, however I could not use self.idata.posterior because mypy 🙈